app/frontend/src/components/Datepicker.tsx

```diff
@@ -41,15 +41,14 @@ export default function Datepicker({
     <MuiPickersUtilsProvider utils={DateFnsUtils}>
       <Controller
         control={control}
-        defaultValue={null}
+        defaultValue={Date()}
```

Comment:
Tests are failing, I think because this returns a number not a date. Try `new Date()`

Reply:
I really need to get better at testing - I'm not sure if it makes a difference but previously default was `minDate`

Reply:
seems to have passed

Reply:
You can run `yarn test` or `yarn test filename` to run the tests locally